index.html

```diff
@@ -23,13 +24,40 @@
         <div class="mode">
             <p class="mode__type">MODE: Who is this character?</p>
             <div class="mode__rules">
-                <p class="mode__title"><i class="fas fa-graduation-cap"></i> Mode rules</p>
+                <p class="mode__title"><i class="fas fa-graduation-cap"></i><span> Mode rules</span></p>
```

Comment:
słabo to trochę wygląda że przy przełączaniu na rules 'MODE: Who is this character?' skacze do góry - może jakieś position absolute?

Reply:
zauważcie że całe tło przeskakuje... a dlaczego? bo na body padding jest ustawiony % zamiast rem :) zmieńcie na 1rem i przestaje skakac tło, zostaje skakanie elementu, które może też skacze przez % gdzieś w margin/padding

Reply:
zmieniłam w body padding na 1rem i nadal przeskakuje :(